### ISSUE TITLE
Save screenshot not correct

### ISSUE THREAD

**Issue by GabrielGLK** [marcomusy/vtkplotter#69]
Hi Marcomusy,

Thank you so much for your vtkplotter module. 
I found the screenshot png size is not correct, the object seems not in the centre of the screenshot and be changed.
Could I output vector figure using vtkplotter?
Thank you so much.

Longkai

**Comment by marcomusy** [comment]
Hi Longkai
on some systems the screenshot function seems to behave a bit weird. Try adding
```python
from vtkplotter import settings
settings.screeshotLargeImage = True
settings.screeshotScale = 1
```
ps: vtk to my knowledge does not really support vector figures

**Comment by GabrielGLK** [comment]
Thank you marcomusy, I tried your solution not worked. The details of the problems as follows(figure attached)
![offscreen_true](https://user-images.githubusercontent.com/39738392/66750392-a1450d00-ee8c-11e9-8e40-1e7a6a4ab384.png)  offscreen = True
![offscreen_false](https://user-images.githubusercontent.com/39738392/66750412-b15cec80-ee8c-11e9-9c56-a537ed00417d.png) offscreen = False
[Selection_019.pdf](https://github.com/marcomusy/vtkplotter/files/3724391/Selection_019.pdf) This one is I cut the section using Shutter, it seems a higher quality figure than 'screenshot' function, I also tried to set (size = (2000.2000)) or settings.screeshotScale = 10(This makes the file very large) in vtkplotter but not work.  
I hope I describe my problem clearly, if not, please tell me.
Thank you.

Longkai

**Comment by marcomusy** [comment]
What is "Shutter"?
I would try to use a smaller window size. What happens if you set
```
settings.screeshotScale = 1
settings.screeshotLargeImage = True
Plotter(size=(500,500))
...
```

**Comment by GabrielGLK** [comment]
Hi, Shutter is just shortcut screenshot tool.
There is blank png I got if set following you. However, I thought this is not the problem of vtkplotter, because I tried some examples also get the same results. 
If I move the figure in renderwindow, then save the screenshot. It seems the figure is limited to top-left of renderwindow and shrunk with a ratio in height level.
![Selection_020](https://user-images.githubusercontent.com/39738392/66762285-fccfc480-eea5-11e9-9f2a-03acdf314ccb.png)
![geometry_1](https://user-images.githubusercontent.com/39738392/66762293-01947880-eea6-11e9-8043-67fabf304807.png)
